static/app/components/group/sentryAppExternalIssueActions.spec.tsx

```diff
@@ -1,6 +1,9 @@
 import {Event as EventFixture} from 'sentry-fixture/event';
+import {Group as GroupFixture} from 'sentry-fixture/group';
 import {Organization} from 'sentry-fixture/organization';
+import {PlatformExternalIssue as PlatformExternalIssueFixture} from 'sentry-fixture/platformExternalIssue';
 import {SentryApp} from 'sentry-fixture/sentryApp';
+import {SentryAppComponent as SentryAppComponentFixture} from 'sentry-fixture/sentryAppComponent';
```

Comment:
Noticing we're inconsistently importing these with different names. Should we do something about it? Should we just rename all the fixtures to be suffixed with `Fixture` in their export

Reply:
Probably yeah

Reply:
Oh yeah it's outlined here as planned: https://github.com/getsentry/frontend-tsc/issues/49